src/parallel-rustc.md

```diff
@@ -10,15 +10,27 @@ These next few sections describe where and how parallelism is currently used,
 and the current status of making parallel compilation the default in `rustc`.
 
 The underlying thread-safe data-structures used in the parallel compiler 
-can be found in `rustc_data_structures/sync.rs`. Some of these data structures
-use the `parking_lot` API.
+can be found in the `rustc_data_structures::sync` module. Some of these data structures
+use the `parking_lot` crate as well.
 
 ## Codegen
 
+There are two underlying thread safe data structures used in code generation.
+The crates they are used in are listed below:
```

Comment:
I don't think listing the crates that use the types is very helpful; and it will also never be fully up to date. It's better to document the types themselves.